plutus-chain-index/README_alpha-release_Marconi.md

```diff
@@ -0,0 +1,75 @@
+
+Marconi is a lightweight, customizable solution for dApp developers who need to index and query the Cardano blockchain.
+
+## Introduction
+
+Marconi is a blockchain query solution that you can configure for the types of information that you need to index and according to how you need to structure your database schema. Marconi provides a way for Plutus off-chain code to have access to data that has already been stored on-chain by indexing portions of the blockchain. 
+
+Marconi gives you the ability to customize the set of data that is indexed. It provides a library of functionality with which you can do the following:
+* build your own indexers, 
+* work only with the data that you care about for your application, 
+* use whatever storage engine you prefer, and 
+* support only the queries that you need to support.
+
+### Scalability through Customized Solutions for Each Use Case
+
+The philosophy behind Marconi is to maintain scalability by customizing the solution for each use case. You can filter the streams of information to transmit only the data you need for your dApp, minimizing network traffic and database storage. You can also customize database schemas and queries so that the dApps have the most efficient and scalable application-specific API to interact with the data.
+
+### Indexing solution ### 
+
+Any indexing solution must deal with blockchain rollbacks, which can happen as the network achieves consensus. Marconi is designed to deal with rollbacks as elegantly as possible while simultaneously finding an effective balance between memory, disk and CPU usage. 
+
+Managing rollbacks is very simple and fast. Marconi drops the events that were rolled back. No complicated logic is required to undo the projection of the list of events on disk, which Marconi would need if it stored everything on disk as quickly as possible. 
+
+Making `K` configurable makes the design quite scalable. Developers do not usually need to guard themselves against rollbacks by `K` blocks, so they can choose to store only 10 events in memory, for example, thereby sacrificing the speed by which Marconi handles rollbacks (since rollbacks on data that is persisted on disk is more expensive than data that is in memory). 
+
+In the event of a restart, recovery is very simple. Marconi stores only fully confirmed transactions, so there is nothing to do other than to load the state persisted on disk and to follow the blockchain from the corresponding chain-point. 
+
+## Queries
+
+The indexed data is accessible through queries. There are no constraints on the format of queries or on the results. Both are identified by a type variable that the indexer exposes. The user, such as a dApp developer, provides the code for both. 
+
+This is currently the only area where you can customize the indexer by providing functions that handle queries and storage. Note that the queries have to run on both the in-memory data and the data that is stored.
```

Comment:
Isn't that too much information for a README? Especially for an Alpha README release?

What's more this kind of information should belong in the documentation site (like ReadTheDocs), not README. Plus, it's already discussed in the ADR from @raduom, so we should probably wait until the ADR is complete before we can say anything about the design choices.

Reply:
I don't think too much information is a bad thing. I think no one knows what Marconi is, so the more we write about it the better. Worst case scenario we will receive heart-breaking feedback on it.

Reply:
That's exactly the challenge of this Alpha release -- for it to be succinct. When there is meaningful brief information that seems like it could be useful for readers to know, I'm including it. At the same time, there is quite a bit that I'm excluding.

Reply:
:+1: